docs/future/compatibility.md

```diff
@@ -0,0 +1,184 @@
+# Enzyme Adapter & Compatibility Proposal
+
+
+## Motivation
+
+This proposal is attempting to address a handful of pain points that Enzyme has been
+subject to for quite a while.  This proposal has resulted mostly [#715](https://github.com/airbnb/enzyme/issues/715),
```

Comment:
the second sentence seems to be missing some word somewhere, maybe "from" after "mostly"